### PR TITLE
AO3 4581 cache the counts in the wrangling sidebar.

### PR DESCRIPTION
**Pull request by zz9pzza**
https://otwarchive.atlassian.net/browse/AO3-4581

### CODE REVIEW

**Comment by CristinaRO** [comment]
Question for clarification: is this the kind of cache that only expires when the expiration time kicks in? So, in this case, the cached count will only expire every hour for each category?

**Comment by zz9pzza** [comment]
That is correct it will only expire after an hour, The tag wrangling chairs were happy to accept two hours and willing to accept 12 hours.

**Comment by CristinaRO** [comment]
If @LadyOscar has pre-emptively given her 🐹 of approval, this can go in.

File: features/step_definitions/tag_set_steps.rb [hunk 1]
```diff
@@ -140,4 +140,10 @@
 When /^I view the tag set "([^\"]*)"/ do |tagset|
   tagset = OwnedTagSet.find_by_title!(tagset)
   visit tag_set_path(tagset)
-end
\ No newline at end of file
+end
+
+When(/^I flush the wrangling sidebar caches$/) do
```
Comment (sarken) [inline features/step_definitions/tag_set_steps.rb]: This issue appears to be broken on test. When you submit a fix for it, can you please move this step definition into the proper file? Right now it's in the tag _set_ steps, but it's not related to tag sets -- it should be in https://github.com/otwcode/otwarchive/blob/master/features/step_definitions/tag_steps.rb, which is where wrangling stuff lives. Thanks!

**Pull request merged**